### ISSUE TITLE
APSIM doesn't work when sharing via webex

### ISSUE THREAD

**Issue by alisonml** [APSIMInitiative/ApsimX#5985]
I'm working on APSIM simulations with international colleagues. When I share my screen in webex to show them the simulation I can no longer expand the 'plus-boxes' at the side of the APSIM tree structure, and I can't use the mouse to scroll down within a page. Instead I have to un-share, make cosmetic changes to the simulation layout and then re-share my screen.

**Comment by hol430** [comment]
Very interesting. @alisonml does it work for you if you share only apsim, as opposed to sharing your whole screen? When I tried sharing my whole screen I get the same behaviour as you - but if I only share apsim, it appears to be working ok. Or rather, I didn't notice any obvious problems.

I'll do some digging and see if we can get a proper fix, but that might serve as an interim solution.

**Comment by alisonml** [comment]
Thanks, @hol430 . I am sharing my whole screen; in future I will try sharing just APSIM (although this is not an idea solution as it means if I also want to share other documents I'll need to un-share and re-share). Thanks for looking for a proper fix!

**Comment by hol430** [comment]
Ok, an alternative workaround is to join the meeting from a web browser rather than from the desktop client. It's not really ideal but may or may not be more convenient than only sharing a single application.

@hol353 - this appears to be a bug in gtk, as I've reproduced the problem in a standalone gtk app. The bug appears to have been fixed, but only in recent versions of gtk3. This fix doesn't appear to have made it into the native libraries used by GtkSharp, although I suppose that may change by the time we actually make the move to gtk3.
